apps/sms/js/thread_ui.js

```diff
@@ -2490,7 +2490,9 @@ var ThreadUI = {
         message: message,
         onerror: function onError(error) {
           var errorName = error.name;
-          this.showMessageSendingError(errorName);
+          // Only sms will have FDN error
+          var recipient = error.data.type === 'sms' && [error.data.receiver];
+          this.showMessageSendingError(errorName, {recipients: recipient});
```

Comment:
You're right that it won't hurt if we set the recipients even the error dialog don't need this. I just sync the behavior in send message but I'm fine to make this part simpler.